frontend/src/metabase/dashboard/components/DashboardHeader.jsx

```diff
@@ -300,19 +302,20 @@ export default class DashboardHeader extends Component {
 
     if (extraButtons.length > 0 && !isEditing) {
       buttons.push(
-        <PopoverWithTrigger
-          triggerElement={
-            <DashboardHeaderButton>
+        <TippyPopoverWithTrigger
+          popoverProps={POPOVER_PROPS}
+          trigger={({ onClick }) => (
```

Comment:
`DashboardHeaderButton` is a `button` element. The old `PopoverWithTrigger` wraps every trigger in an anchor tag, so this button ended up looking like `<a><button>...</button></a>` which is just wrong. Here, I'm using `TippyPopoverWithTrigger`'s replacement for `triggerElement` -- `trigger` -- which lets you totally control what gets used as the trigger if you give it a function.